README.md

```diff
@@ -0,0 +1,33 @@
+**THIS BRANCH WORKS WITH HTML INSTEAD OF RMD FILES AS INPUT**
+
+# Convert HTML to Redmine (Textile) format
+
+This script aims to near-automate Redmine updates.
+
+
+## How to use:
+
+Navigate to the directory containing the RMarkdown file to convert. Make sure rmd2redmine.sh is in the same directory. Then use the system terminal to execute:
+
+```bash
+$ ./rmd2redmine.sh -f <HTML file>
+```
+
+Upon invocation, the script extracts the figures, saves them as png files and finally combines it all to a `.redmine` output file.
+
+Copy the contents of the `.redmine` file straight into your Redmine update, upload the automatically generated png figures and you're done.
+
+
+### Working with figures
+
+Figures are handled automatically. You don't even need to save them manually.
```

Comment:
Maybe add information that you have to upload them on redmine though. I would consider adding the date both to the .png folder you create and every single .png file, just that people do not on accident overwrite older posts and figures they might have created in the same folder.